dockers/docker-snmp-sv2/Dockerfile.j2

```diff
@@ -22,9 +25,9 @@ ENV DEBIAN_FRONTEND=noninteractive
 # TODO: remove libpython3.6-dev, its and pip's dependencies if we can get pip3 directly
 # install subagent
 # clean up
-RUN apt-get update && apt-get install -y libperl5.20 libpci3 libwrap0 \
-            libexpat1-dev \
-            curl gcc && \
+RUN apt-get update && apt-get install -y --force-yes libperl5.20 libpci3 libwrap0 \
```

Comment:
why do we need force-yes?

Reply:
It was required by python click default group but I will double check and remove if no needed.

Reply:
@lguohan 
It was required by following packets:
WARNING: The following packages cannot be authenticated!
  python-click python-click-default-group python-tabulate
E: There are problems and -y was used without --force-yes